deployment/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ x-defaults: &defaults
   env_file: services.env
   volumes:
     - /data/state:/state
-    - ~/keystore:/keystore
+    - /data/keystore:/keystore
```

Comment:
One suggestion to make this more portable would be to only put a default here that works for local development but is overridable via a variable in `.env` ([example](https://github.com/raiden-network/raiden-transport/blob/b825292844c67327873aaa8231228774d8866015/docker-compose.yml#L16)).